src/sage/libs/gap/element.pyx

```diff
@@ -1702,8 +1716,17 @@ cdef class GapElement_IntegerMod(GapElement):
             # ring = self.DefaultRing().sage()
             characteristic = self.Characteristic().sage()
             ring = ZZ.quotient_ring(characteristic)
-        return self.lift().sage(ring=ring)
+        return ring(self.lift())
 
+    def _integer_(self, R):
+        r"""
+        TESTS::
+
+            sage: n = libgap.eval('One(ZmodnZ(123)) * 13')
```

Comment:
You could also do this:
```suggestion
            sage: n = libgap.eval('ZmodnZObj(13, 123))')
```
or, if libgap makes it easy to directly call GAP functions with small int arguments, then something like this:
```suggestion
            sage: n = libgap.ZmodnZObj(13, 123)
```